libs/wingc/src/jsify.rs

```diff
@@ -1280,10 +1295,9 @@ impl<'a> JSifier<'a> {
 		code.open(format!("require(\"{client_path}\")({{"));
 
 		if let Some(lifts) = &ctx.lifts {
-			for capture in lifts.captures() {
-				let preflight = capture.code.clone();
-				let lift_type = format!("context._lift({})", preflight);
-				code.line(format!("{}: ${{{}}},", capture.token, lift_type));
+			for (token, capture) in lifts.captures.iter().filter(|(_, cap)| !cap.is_field) {
+				let lift_type = format!("context._lift({})", capture.code);
+				code.line(format!("{}: ${{{}}},", token, lift_type));
```

Comment:
How come fields don't have to be lifted?

Reply:
This creates the `_toInflightType` which lifts free floating vars (and types) used by the inflight (client) type. 
Fields, on the other hand, are lifted by `_toInfilght` when it calls the constructor of the type returned by `_toInflightType`.
I'm not totally sure this complexity of separating the fields from the free floating vars, and doing so in this way, is really required. But I didn't change the previous logic here (previously the `lifts.captures` map didn't include all captures just non-field ones, I unified it).